main.go

```diff
@@ -1,29 +1,35 @@
 package main
 
-import "flag"
+import (
+	"flag"
+	"fmt"
+	"os"
+)
 
 func main() {
 	const (
-		torrentDescription = "Required: path of the torrent file."
-		outDescription     = "Optional: path of the output file.\nIf not set, the file will be downloaded in the same folder as the torrent file with the name in that file."
+		outDescription = "Optional: path of the output file.\nIf not set, the file will be downloaded in the same folder as the torrent file with the name in that file."
 	)
-	var torrentPath string
-	var outPath string
-
-	flag.StringVar(&torrentPath, "f", "", torrentDescription)
-	flag.StringVar(&torrentPath, "file", "", torrentDescription)
 
-	flag.StringVar(&outPath, "o", "", outDescription)
-	flag.StringVar(&outPath, "output", "", outDescription)
+	args := os.Args
 
-	flag.Parse()
-
-	if torrentPath == "" {
+	if len(args) <= 1 {
 		println("Please provide a path for the torrent file")
 		return
 	}
 
-	err := Download(torrentPath, outPath)
+	torrentPath := args[1]
```

Comment:
this means that you can't do go-torrent -o foo mytorrent-file.torrent

Reply:
Hmmm, the code is broken; `flag` does not allow positional arguments to appear before flags...

Reply:
yeah, like the go binary itself.

is hand-parsing os.Args that big of a deal? it seems to be simpler than working around those limitations.

Reply:
see https://github.com/matei-oltean/go-torrent/commit/bb2a335255329c43dd929972e2afdff9889f92d6

Reply:
It seems better indeed. We would lose `-o=abcd` arguments though.